.github/workflows/release.yml

```diff
@@ -97,14 +97,20 @@ jobs:
         uses: actions/attest-build-provenance@v1
         with:
           subject-path: |
-            dist/pex
+            dist/pex*
             dist/docs/pdf/pex.pdf
       - name: Prepare Changelog
         id: prepare-changelog
         uses: a-scie/actions/changelog@v1.6
         with:
           changelog-file: ${{ github.workspace }}/CHANGES.md
           version: ${{ needs.determine-tag.outputs.release-version }}
+      - name: Append Hashes to Changelog
+        run: |
+          changelog_tmp="$(mktemp)"
+          cat "${{ steps.prepare-changelog.outputs.changelog-file }}" <(echo '***') dist/hashes.md \
+            > "${changelog_tmp}"
+          mv "${changelog_tmp}" "${{ steps.prepare-changelog.outputs.changelog-file }}"
```

Comment:
The upshot of this is the Releases will get this style content between the change-log and the files list below:
***
|file|sha256|size|
|----|------|----|
|pex|b5fb8b2469dd8f208a7805c7ec51850e81f4a4952e4808827b8e07589bc2496f|4306002|
|pex-linux-aarch64|1861382d2c273105a4b7013054e7acbab02f9ce78de7baa7d5e744c398eb4204|24854091|
|pex-linux-x86_64|e42f3231f605147ccb93118c2bb1386ca95dfac624b19cb405e91db9886cd7ad|28404512|
|pex-macos-aarch64|b1f18587833c9af437449dddf53dc778b1705eaace750b0f02ecee3bba5d2103|22320777|
|pex-macos-x86_64|fb8e9c580a9dc7ed0aae9f5b4ecad44287d4db585980400c2c3893687e545e74|22816650|